appinfo/routes.php

```diff
@@ -65,7 +65,12 @@
 		['name' => 'card#removeLabel', 'url' => '/cards/{cardId}/label/{labelId}', 'verb' => 'DELETE'],
 		['name' => 'card#assignUser', 'url' => '/cards/{cardId}/assign', 'verb' => 'POST'],
 		['name' => 'card#unassignUser', 'url' => '/cards/{cardId}/unassign', 'verb' => 'PUT'],
+		
+		// dashboard
+		['name' => 'card#findAllWithDue', 'url' => '/dashboard/due', 'verb' => 'GET'],
```

Comment:
Please move this to an ocs route and use an OCSController. That way we don't need separate api routes and external apps can use the same ones that the web UI uses.